maven-plugin/src/test/resources/integration-test/invokeIT/pom.xml

```diff
@@ -0,0 +1,78 @@
+<project
+        xmlns="http://maven.apache.org/POM/4.0.0"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                      http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.openmrs</groupId>
+    <artifactId>openmrs-sdk-integration-test-project</artifactId>
+    <version>1.0</version>
+    <packaging>pom</packaging>
+
+    <!--
+        This makes the test classes defined within this project available on the Plugin classpath when it is executed
+        Any new test libraries that are referenced by tests that are use this pom (eg. those that extend AbstractMavenIT)
+        need to be added as dependencies must be added here so the plugin can load and execute the tests successfully
+        See:  https://maven.apache.org/guides/mini/guide-maven-classloading.html
+    -->
```

Comment:
This took me the longest time to figure out, but finally was able to do so.  See comment above.